samples/snippets/snippets_mute_config.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START securitycenter_create_mute_config]
+from google.cloud.securitycenter_v1 import Finding
+
+
+def create_mute_rule(parent_path: str, mute_config_id: str) -> None:
```

Comment:
```suggestion
from google.cloud.securitycenter_v1 import Finding


# [START securitycenter_create_mute_config]
def create_mute_rule(parent_path: str, mute_config_id: str) -> None:
```

Reply:
In addtion, should we move the `from google.cloud.securitycenter_v1 import Finding` import to the `set_mute_unmute_finding` sample for consistency?